db/schema.rb

```diff
@@ -33,6 +33,39 @@
   add_index "authentications", ["uid", "provider"], :name => "index_authentications_on_uid_and_provider", :unique => true
   add_index "authentications", ["user_id", "provider"], :name => "index_authentications_on_user_id_and_provider", :unique => true
 
+  create_table "c_rater_feedback_items", :force => true do |t|
+    t.text     "answer_text"
+    t.integer  "answer_id"
+    t.string   "answer_type"
+    t.string   "item_id"
+    t.string   "status"
+    t.integer  "score"
+    t.text     "feedback_text"
+    t.text     "response_info"
+    t.datetime "created_at",    :null => false
+    t.datetime "updated_at",    :null => false
+  end
+
+  add_index "c_rater_feedback_items", ["answer_id", "answer_type"], :name => "c_rat_feed_it_answer_idx"
+
+  create_table "c_rater_score_mappings", :force => true do |t|
+    t.text     "mapping"
+    t.datetime "created_at", :null => false
+    t.datetime "updated_at", :null => false
+  end
+
+  create_table "c_rater_settings", :force => true do |t|
```

Comment:
I think it would be better for this model to have a name like c_rater_item (or replace 'item' with whatever the c-rater folks call their items). Or c_rater_item_settings. The use of just c_rater_settings made me think these were settings for the global interaction with c_rater.